dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/StatisticsLoadBalance.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.dubbo.rpc.cluster.loadbalance;
+
+import org.apache.dubbo.common.URL;
+import org.apache.dubbo.monitor.dubbo.CpuUsageService;
+import org.apache.dubbo.monitor.dubbo.CpuUsageServiceImpl;
+import org.apache.dubbo.rpc.Invocation;
+import org.apache.dubbo.rpc.Invoker;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StatisticsLoadBalance extends AbstractLoadBalance {
+
+    private Map<String, Float> cpuUsage = new HashMap<>();
+
+    public StatisticsLoadBalance() {
+        CpuUsageService cpuUsageService = new CpuUsageServiceImpl(100L, 500L);
```

Comment:
@ralf0131 Should I add these parameters as properties that could change in this PR?

Reply:
Hi, I think they should be configurable.